src/main/java/org/datadog/jenkins/plugins/datadog/tracer/JavaConfigurator.java

```diff
@@ -94,7 +97,12 @@ private byte[] getTracerSignaturePublicKey(DatadogTracerJobProperty<?> tracerCon
     private <T> T getSetting(DatadogTracerJobProperty<?> tracerConfig, String envVariableName, T defaultValue, Function<String, T> parser) {
         String envVariable = getEnvVariable(tracerConfig, envVariableName);
         if (envVariable != null) {
-            return parser.apply(envVariable);
+            try {
+                return parser.apply(envVariable);
+            } catch (Exception e) {
+                LOGGER.fine("Variable " + envVariableName + " has value " + envVariable + " which could not be parsed. " +
```

Comment:
Could we avoid logging the value of the env var to avoid secret leaks?

Reply:
Good point, changed it to not log the values.